libvast/src/pipeline.cpp

```diff
@@ -21,6 +23,46 @@
 
 namespace vast {
 
+caf::expected<pipeline>
+pipeline::parse(std::string name, std::string_view repr) {
+  auto result = pipeline{std::move(name), {}};
+  // plugin name parser
+  using parsers::alnum, parsers::chr, parsers::space;
```

Comment:
Should we use blank over space here?

Reply:
Replaced in https://github.com/tenzir/vast/pull/2877/commits/92595e7450a00df6fb03e8c9458d6aec21a8e58c